spec/requests/automate_workspaces_spec.rb

```diff
@@ -40,6 +41,15 @@
       expect(response).to have_http_status(:ok)
     end
 
+    it 'should mask password attributes' do
+      api_basic_authorize action_identifier(:automate_workspaces, :read, :resource_actions, :get)
+      get(api_automate_workspace_url(nil, aw.guid))
+
+      expect(response).to have_http_status(:ok)
+      expect(response.parsed_body['input']['objects']['root']['var2']).to eq(masked_password)
+      expect(response.parsed_body['input']['method_parameters']['arg1']).to eq(masked_password)
```

Comment:
I'd prefer replacing 49-50 with expect(response.parsed_body).to include('input' => a_hash_including( ... )